app/Main.hs

```diff
@@ -22,16 +23,18 @@ main = do
     let configFile = case args of
             [cnf] -> cnf
             _ -> error usage
-    run configFile repostUpdates
+    run configFile
   where
```

Comment:
I didn't understand what's the point of this change. In previous version, `run` was responsible for unpacking the transformer stack upto the base monad. In your version, it runs the whole transformed monad. Both semantics are correct, but they are simply two different semantics.